.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,58 +4,43 @@ about: Create a report to help us improve existing features
 
 ---
 
-<!--
-DELETE ME
+# Checklist
```

Comment:
To be honest, this feels a bit pedantic. I'd turn it into a simple list "Please: 1 2" I think it's better to integrate the URL as a 'field' in the issue template.

Reply:
I have a feeling that it will not be read if you do not have to interact with it. The list could end up like that wall of text that we had earlier. You can see checkboxes like those in some other projects, too.